doc/Language/traps.pod6

```diff
@@ -530,7 +530,7 @@ in the list, which will be numified separately.
 In some languages one could reference the last element of an array by
 asking for the "-1th" element of the array, e.g.:
 
-=for code :lang<perl5>
+=for code :lang<Perl>
```

Comment:
You've used lower-case perl elsewhere. You should probably stick to that.

Reply:
Fixed: 1291bd1